package.json

```diff
@@ -12,8 +12,9 @@
     "postinstall": "npm prune && npm run build",
     "prepush": "opt --out prepush --exec 'npm run validate'",
     "start": "webpack-dev-server --config webpack/starterApp.js --devtool eval --progress --colors --hot --history-api-fallback --open",
-    "test": "NODE_ENV=test karma start",
-    "test:watch": "npm test -- --auto-watch --no-single-run --reporters progress",
+    "test": "jest",
+    "test:watch": "npm test -- --verbose --notify --watch",
```

Comment:
How do you make this work in a 4 or 5 line test window at the bottom of the editor?  Or do you not do it that way?

Wondering if there's a way we could make this a personal preference thing?